docs/static/[version]/examples.js

```diff
@@ -286,9 +286,20 @@ async function displayChart(
   if (!mode) {
     mode = 'default';
   }
+
+  if (!cssThemeName) {
+    cssThemeName = 'BOOSTED5';
+  }
+  let iframe = document.querySelector(`#${id} iframe`);
+  let clientHeight = 0;
+  if (iframe) {
```

Comment:
This makes the charts window reload each time, it seems unpleasant to me at least is that normal ?